specs/TrackingPrevention.md

```diff
@@ -0,0 +1,206 @@
+Tracking Prevention
+===
+
+# Background
+The WebView2 team has been asked for an API to toggle tracking prevention and also ability to control levels
+of tracking prevention.
+
+We are proposing two API's
+IsTrackingPreventionEnabled: This API allows you to enable/disable the tracking prevention feature. By default
+tracking prevention feature is enabled and set to `Balanced` level for WebView2. You can set the
+`CoreWebView2EnvironmentOptions.IsTrackingPreventionEnabled` property to false to disable the tracking prevention
+feature for WebView2 before creating environment that also skips the related code and improves the performance.
+
+TrackingPreventionUserPreference: This API allows you to control levels of tracking prevention for WebView2 which
```

Comment:
This API name has changed. Please review this background section if anything else needs to be updated.